src/main/java/com/asu/cse545/group12/services/AuthorizationServiceImpl.java

```diff
@@ -136,31 +157,32 @@ else if (Const.TRANSFER_REQUEST.equals(authorization.getRequestType())) {
 			}
 
 		}
-		return authorizationDao.approve(authorization);
+		//return authorizationDao.approve(authorization);
+		return 0;
 	}
 
 	@Override
 	public List<Authorization> getNotifications(Users user) {
-		// TODO Auto-generated method stub
-		if("1".equals(user.getRoleId()))
+		// TODO Auto-generated method stub 
+		if("1".equals(user.getRoleId().toString()))
```

Comment:
Both are same na... 

-----Original Message-----
From: "Nagarjuna Myla" <notifications@github.com>
Sent: ‎10/‎23/‎2015 6:22 PM
To: "chasethenag420/sbs" <sbs@noreply.github.com>
Cc: "Charan Thej Aware" <charan.1312@gmail.com>
Subject: Re: [sbs] lot changes (#104)

In src/main/java/com/asu/cse545/group12/services/AuthorizationServiceImpl.java:
>  	}
>  
>  	@Override
>  	public List<Authorization> getNotifications(Users user) {
> -		// TODO Auto-generated method stub
> -		if("1".equals(user.getRoleId()))
> +		// TODO Auto-generated method stub 
> +		if("1".equals(user.getRoleId().toString()))
could have use 1==user.getRoleId() instead of converting to string
—
Reply to this email directly or view it on GitHub.